content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
+description = "Migration from kfctl_existing_arrikto.yaml config"
+weight = 4
+page_hide = true
++++
+
+If you were using the `kfctl_existing_arrikto` configuration in Kubeflow v0.7 or earlier, you should use {{% config-uri-istio-dex %}} in Kubeflow {{% kf-latest-version %}} following the instructions in the [`kfctl_istio_dex`](/docs/started/k8s/kfctl-istio-dex/) page.
```

Comment:
How about this slight change, for more clarity:

```
If you were using the `kfctl_existing_arrikto` configuration in Kubeflow v0.7 or earlier, you should use {{% config-file-istio-dex %}} in Kubeflow {{% kf-latest-version %}}. Follow the instructions in the [guide to `kfctl_istio_dex`](/docs/started/k8s/kfctl-istio-dex/).
```

(I've changed the wording of the last part of the sentence, and also change the URI to just the file name in the variable.)

Reply:
Sure

Reply:
Thanks!